### PR TITLE
remove debug message from MemoryIOStream

### PR DESCRIPTION
**Pull request by urshanselmann**
removes a debug message that leaks to stdout when importing a file using Importer::ReadFileFromMemory

### CODE REVIEW

**Comment by Jackie9527** [comment]
https://github.com/assimp/assimp/blob/6ed433d011dd95d7afc297e7836c1f9df96d6cf9/code/AssetLib/Ogre/OgreXmlSerializer.cpp#L493

Another debug message, please remove together.

**Comment by urshanselmann** [comment]
> https://github.com/assimp/assimp/blob/6ed433d011dd95d7afc297e7836c1f9df96d6cf9/code/AssetLib/Ogre/OgreXmlSerializer.cpp#L493
> 
> Another debug message, please remove together or replace it with log.

Done. 

Also added a simple shell script to search for those in future pull requests (see https://github.com/assimp/assimp/actions/runs/4331295658/jobs/7563038137)

File: code/AssetLib/Ogre/OgreXmlSerializer.cpp [hunk 1]
```diff
@@ -490,7 +490,7 @@ bool OgreXmlSerializer::ImportSkeleton(Assimp::IOSystem *pIOHandler, MeshXml *me
     OgreXmlSerializer serializer(xmlParser.get());
     XmlNode root = xmlParser->getRootNode();
     if (std::string(root.name()) != nnSkeleton) {
-        printf("\nSkeleton is not a valid root: %s\n", root.name());
+        // printf("\nSkeleton is not a valid root: %s\n", root.name());
```
Comment (turol) [inline code/AssetLib/Ogre/OgreXmlSerializer.cpp:493]: Commenting it out is not removing it. And this really should be a log message.

Comment (urshanselmann) [reply code/AssetLib/Ogre/OgreXmlSerializer.cpp:493]: I am not familiar with the context of this log message, so leaving it as a comment for the next person who works on this seemed like the most reasonable thing to do.

If you happen to be familiar with it, please let me know whether I should remove it altogether, or which logging function/macro to use. Thanks!

**Review by kimkulling — APPROVED** [review]

**Comment by kimkulling** [comment]
Merged, thanks a lot for your contribution.

**Pull request merged**